src/Integration.UnitTests/Binding/BindCommandTests.cs

```diff
@@ -37,6 +37,7 @@ public class BindCommandTests
         [TestInitialize]
         public void TestInitialize()
         {
+            KnownUIContextsAccessor.Reset();
```

Comment:
There was a test bug in the KnownUIContextsAccessor that I fixed, was found since KnownUIContexts are used in two places now.